From 5338c9c323944d555d0fed24be52752fd5e8cf35 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 3 Nov 2015 01:09:20 +0100
Subject: [PATCH] ip-rule: neither prohibit nor reject or unreachable flags
 exist

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1243413

commit ccaf6eb5cc1020c7fe8997b8cadf9a12cf92302a
Author: Phil Sutter <phil@nwl.cc>
Date:   Thu Oct 15 22:32:17 2015 +0200

    ip-rule: neither prohibit nor reject or unreachable flags exist

    This has been inconsistent since the beginning of Git and seems to be
    merely a documentation leftover, therefore just remove it from help
    output and man page.

    Signed-off-by: Phil Sutter <phil@nwl.cc>

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 ip/iprule.c   | 1 -
 man/man8/ip.8 | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/ip/iprule.c b/ip/iprule.c
index 29de5fc..639ae83 100644
--- a/ip/iprule.c
+++ b/ip/iprule.c
@@ -36,7 +36,6 @@ static void usage(void)
 	fprintf(stderr, "SELECTOR := [ not ] [ from PREFIX ] [ to PREFIX ] [ tos TOS ] [ fwmark FWMARK[/MASK] ]\n");
 	fprintf(stderr, "            [ dev STRING ] [ pref NUMBER ]\n");
 	fprintf(stderr, "ACTION := [ table TABLE_ID ]\n");
-	fprintf(stderr, "          [ prohibit | reject | unreachable ]\n");
 	fprintf(stderr, "          [ realms [SRCREALM/]DSTREALM ]\n");
 	fprintf(stderr, "          [ goto NUMBER ]\n");
 	fprintf(stderr, "TABLE_ID := [ local | main | default | NUMBER ]\n");
diff --git a/man/man8/ip.8 b/man/man8/ip.8
index 990c6fe..1548836 100644
--- a/man/man8/ip.8
+++ b/man/man8/ip.8
@@ -273,7 +273,7 @@ throw " | " unreachable " | " prohibit " | " blackhole " | " nat " ]"
 .IR TABLE_ID " ] [ "
 .B  nat
 .IR ADDRESS " ] [ "
-.BR prohibit " | " reject " | " unreachable " ] [ " realms
+.B realms
 .RI "[" SRCREALM "/]" DSTREALM " ]"
 
 .ti -8
-- 
1.8.3.1

